src/Symfony/Component/Mime/RawMessage.php

```diff
@@ -18,11 +18,14 @@
  */
 class RawMessage
 {
-    /** @var iterable|string|resource */
+    /** @var iterable<string>|string|resource */
     private $message;
     private bool $isGeneratorClosed;
 
-    public function __construct(iterable|string $message)
+    /**
+     * @param iterable<string>|string|resource $message
```

Comment:
This should be fixed in 5.4 as well, right? So, let's move the target to 5.4.

Reply:
RawMessage doesn't support resources in 5.4 it seems